src/form/components/StringBuilder/StringBuilder.test.js

```diff
@@ -1,7 +1,52 @@
+import React from 'react';
+import { mount } from '@cypress/react';
 import StringBuilder from './index';
+import '../../../index.scss';
 
-describe('StringBuilder', () => {
-  it('TODO', () => {
-    expect(true).to.equal(true);
+describe('Button', () => {
```

Comment:
why Button?

Reply:
nice catch, changing it .. had copied the boiler plate from button's test 🤦